site/code-snippets/web5/build/decentralized-web-nodes/ update-dwn.js

```diff
@@ -0,0 +1,9 @@
+export async function updateDWN
+const { record } = await web5.dwn.records.read({
+  message: {
+    recordId: 'bafyreiaz5oycqbrnmmpvffxqyoqxvx6bcnqueprmt2qnvzcurpc52r5uyy',
+  }
+});
+
+const status = await record.update({ data: "Hello, I'm updated!" });
+```
```

Comment:
```suggestion
return status;
}
```

- removed the leftover code block syntax that was copied over
- added a return statement. This will be needed for the test. In the test, you'll need to make sure that `status.code` is the correct value, indicating that the update was successful and the code snippet works.
- added closing curly brace for the function